docs/README.md

```diff
@@ -14,7 +14,7 @@ cascade:
 <!-- prettier-ignore end -->
 
 
-# Overview of Tekton Triggers
+# Tekton Triggers
 
 Tekton Triggers is a Tekton component that allows you to detect and extract information from events from a variety of sources and deterministically instantiate
 and execute [`TaskRuns`](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md) and [`PipelineRuns`](https://github.com/tektoncd/pipeline/blob/master/docs/pipelineruns.md)
```

Comment:
While in the process of adapting docs for the website: is there a reason for using absolute links here? As can be seen in the [deploy-preview](https://deploy-preview-270--tekton.netlify.app/docs/triggers/) of tektoncd/website#270, this means clicking the links while on the website will lead to GitHub instead of the corresponding page on the website. 

How about changing these to relative links?

Reply:
Yeah, relative links would be better. Nice catch. Thanks!